recipes/ragel/all/CMakeLists.txt

```diff
@@ -1,133 +1,131 @@
-	source_subfolder/ragel/goflat.cpp
-	source_subfolder/ragel/gofflat.cpp
-	source_subfolder/ragel/gogoto.cpp
-	source_subfolder/ragel/gofgoto.cpp
-	source_subfolder/ragel/goipgoto.cpp
-	source_subfolder/ragel/gotablish.cpp
-	source_subfolder/ragel/mlcodegen.cpp
-	source_subfolder/ragel/mltable.cpp
-	source_subfolder/ragel/mlftable.cpp
-	source_subfolder/ragel/mlflat.cpp
-	source_subfolder/ragel/mlfflat.cpp
-	source_subfolder/ragel/mlgoto.cpp
-	source_subfolder/ragel/mlfgoto.cpp
+	${RAGEL_SOURCE_DIR}/ragel/main.cpp
+	${RAGEL_SOURCE_DIR}/ragel/parsetree.cpp
+	${RAGEL_SOURCE_DIR}/ragel/parsedata.cpp
+	${RAGEL_SOURCE_DIR}/ragel/fsmstate.cpp
+	${RAGEL_SOURCE_DIR}/ragel/fsmbase.cpp
+	${RAGEL_SOURCE_DIR}/ragel/fsmattach.cpp
+	${RAGEL_SOURCE_DIR}/ragel/fsmmin.cpp
+	${RAGEL_SOURCE_DIR}/ragel/fsmgraph.cpp
+	${RAGEL_SOURCE_DIR}/ragel/fsmap.cpp
+	${RAGEL_SOURCE_DIR}/ragel/rlscan.cpp
+	${RAGEL_SOURCE_DIR}/ragel/rlparse.cpp
+	${RAGEL_SOURCE_DIR}/ragel/inputdata.cpp
+	${RAGEL_SOURCE_DIR}/ragel/common.cpp
+	${RAGEL_SOURCE_DIR}/ragel/redfsm.cpp
+	${RAGEL_SOURCE_DIR}/ragel/gendata.cpp
+	${RAGEL_SOURCE_DIR}/ragel/cdcodegen.cpp
+	${RAGEL_SOURCE_DIR}/ragel/cdtable.cpp
+	${RAGEL_SOURCE_DIR}/ragel/cdftable.cpp
+	${RAGEL_SOURCE_DIR}/ragel/cdflat.cpp
+	${RAGEL_SOURCE_DIR}/ragel/cdfflat.cpp
+	${RAGEL_SOURCE_DIR}/ragel/cdgoto.cpp
+	${RAGEL_SOURCE_DIR}/ragel/cdfgoto.cpp
+	${RAGEL_SOURCE_DIR}/ragel/cdipgoto.cpp
+	${RAGEL_SOURCE_DIR}/ragel/cdsplit.cpp
+	${RAGEL_SOURCE_DIR}/ragel/javacodegen.cpp
+	${RAGEL_SOURCE_DIR}/ragel/rubycodegen.cpp
+	${RAGEL_SOURCE_DIR}/ragel/rubytable.cpp
+	${RAGEL_SOURCE_DIR}/ragel/rubyftable.cpp
+	${RAGEL_SOURCE_DIR}/ragel/rubyflat.cpp
+	${RAGEL_SOURCE_DIR}/ragel/rubyfflat.cpp
+	${RAGEL_SOURCE_DIR}/ragel/rbxgoto.cpp
+	${RAGEL_SOURCE_DIR}/ragel/cscodegen.cpp
+	${RAGEL_SOURCE_DIR}/ragel/cstable.cpp
+	${RAGEL_SOURCE_DIR}/ragel/csftable.cpp
+	${RAGEL_SOURCE_DIR}/ragel/csflat.cpp
+	${RAGEL_SOURCE_DIR}/ragel/csfflat.cpp
+	${RAGEL_SOURCE_DIR}/ragel/csgoto.cpp
+	${RAGEL_SOURCE_DIR}/ragel/csfgoto.cpp
+	${RAGEL_SOURCE_DIR}/ragel/csipgoto.cpp
+	${RAGEL_SOURCE_DIR}/ragel/cssplit.cpp
+	${RAGEL_SOURCE_DIR}/ragel/dotcodegen.cpp
+	${RAGEL_SOURCE_DIR}/ragel/xmlcodegen.cpp
+	${RAGEL_SOURCE_DIR}/ragel/gocodegen.cpp
+	${RAGEL_SOURCE_DIR}/ragel/gotable.cpp
+	${RAGEL_SOURCE_DIR}/ragel/goftable.cpp
+	${RAGEL_SOURCE_DIR}/ragel/goflat.cpp
+	${RAGEL_SOURCE_DIR}/ragel/gofflat.cpp
+	${RAGEL_SOURCE_DIR}/ragel/gogoto.cpp
+	${RAGEL_SOURCE_DIR}/ragel/gofgoto.cpp
+	${RAGEL_SOURCE_DIR}/ragel/goipgoto.cpp
+	${RAGEL_SOURCE_DIR}/ragel/gotablish.cpp
+	${RAGEL_SOURCE_DIR}/ragel/mlcodegen.cpp
+	${RAGEL_SOURCE_DIR}/ragel/mltable.cpp
+	${RAGEL_SOURCE_DIR}/ragel/mlftable.cpp
+	${RAGEL_SOURCE_DIR}/ragel/mlflat.cpp
+	${RAGEL_SOURCE_DIR}/ragel/mlfflat.cpp
+	${RAGEL_SOURCE_DIR}/ragel/mlgoto.cpp
+	${RAGEL_SOURCE_DIR}/ragel/mlfgoto.cpp
 )
 
 add_executable(${CMAKE_PROJECT_NAME} ${SRC} ${HEADERS})
 target_link_libraries(${CMAKE_PROJECT_NAME} PUBLIC ${CONAN_LIBS})
```

Comment:
```suggestion
```

CONAN_LIBS no longer works with CMakeToolchain, and, ragel does not have requirements.